index.html

```diff
@@ -31,14 +31,82 @@ <h3>I’am a Software Developer</h3>
+        <p class="par-black">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's
+            standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div>
+            <img src="pictures/Tag.png" alt="">
+            <img src="pictures/Tag(1).png" alt="">
+            <img src="pictures/Tag(2).png" alt="">
+            <img src="pictures/Tag(3).png" alt="">
+        </div>
+        <img src="pictures/Enable.png" class="fiveg" alt="">
+        <section class="activ-g">
+            <h6 class="en">Profesional Art<br>Printing Data <br></h6>
+            <p class="text2">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups
+                required. has been the <br>
+                industry's standard</p>
+            <div class="en">
+                <img src="pictures/translu.png" alt="">
+                <img src="pictures/translu1.png" alt="">
+                <img src="pictures/translu2.png" alt="">
+            </div>
+            <div class="oragan">
+                <button class="button">See Project</button>
+            </div>
+        </section>
+    </section>
+    <section class="laspart">
+        <h2 class="en">About me</h2>
+        <p class="texf en">Hello I’m a software developer! I can help you build a product , feature or website Look through
+            some of my
+            work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+            me.</p>
+        <button class="butm en">Get My Resume</button>
+    </section>
+    <section class="en">
+        <img src="pictures/linew.png" alt="">
+        <div class="d-flex">
+            <div class="constys">
+                <img src="pictures/Rectangle55.png" alt="">
+            </div>
+            <div>
+                <p class="endt">Lenguage</p>
 
+            </div>
+        </div>
+        <ul class="enlase">
+            <li><a href="#">Javascript</a></li>
+            <li><a href="#">Ruby</a></li>
+            <li><a href="#">Html</a></li>
+            <li><a href="#">Css</a></li>
+        </ul>
+        <p class="endt">Frameworks</p>
+        <ul class="enlase">
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby on Rails</a></li>
+            <li><a href="#">RSPec</a></li>
+            <li><a href="#">CapyBara</a></li>
+            <li><a href="#">Selenium</a></li>
+        </ul>
+        <p class="endt">Skills</p>
+        <ul class="enlase">
+            <li><a href="#">Codekit</a></li>
+            <li><a href="#">Github</a></li>
+            <li><a href="#">Codepen</a></li>
+            <li><a href="#">Gitlab</a></li>
+            <li><a href="#">Terminal</a></li>
+        </ul>
+        <a href="https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27">View Project</a>
+    </section>
```

Comment:
- [ ] As mentioned earlier, please use proper `2-space indentation` for your HTML. You may change this value in your Visual Studio Code settings. Check out this youtube video on how to do this: https://www.youtube.com/watch?v=m6gUdoRIRz8